app/views/tasks/show.html.erb

```diff
@@ -19,9 +19,15 @@
           <%= sanitize GovukMarkdown.render(action.guidance_text) %>
         <% end %>
       <% end %>
+    <% end %>
 
+    <% if @task.optional? %>
+      <%= form.govuk_check_box_divider %>
+      <%= form.govuk_check_box :not_applicable,
```

Comment:
Shame the JS isn't working here. I suppose they would need matching keys

Reply:
Yep, I had a very quick think about if we could use the JS, but decided it could wait for now.